driver_test.go

```diff
@@ -1967,6 +1967,34 @@ func TestClientSessionKeepAliveParameter(t *testing.T) {
 	defer rows.Close()
 }
 
+func TestTimePrecision(t *testing.T) {
+	var db *sql.DB
+	var err error
+
+	if db, err = sql.Open("snowflake", dsn); err != nil {
+		t.Fatalf("failed to open db. %v, err: %v", dsn, err)
+	}
+	defer db.Close()
+	err = db.Ping()
```

Comment:
Ping is not required as you run db.Exec soon.

Reply:
Ok, I will remove Ping.